isb_web/manage.py

```diff
@@ -246,6 +256,7 @@ def userinfo(request: starlette.requests.Request):
             "id_token": user.get("id_token"),
             "expires_at": user.get("expires_at"),
             "auth_time": auth_time,
+            "access_token" : user.get("access_token")
```

Comment:
Also expose the access token as we need to use this to get a "throughput token". Using this access token, we are able to exchange it to a "throughput token", which can then be used to create annotations in throughput.